### PR TITLE
Fix Kudu merge error between two CTAS table

### PR DESCRIPTION
**Pull request by chenjian2664**
## Description

Fix https://github.com/trinodb/trino/issues/16848

## Release notes

(x) Release notes are required, with the following suggested text:

```markdown
# Kudu
* Fix Kudu merge error between two tables that created by CTAS. ({issue}`16848`)
```

### CODE REVIEW

File: testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java [hunk 1]
```diff
@@ -5163,6 +5163,36 @@ public void testPotentialDuplicateDereferencePushdown()
...
+                CREATE TABLE %s AS
+                SELECT * FROM (
+                        VALUES
+                        (1, 'a', 'aa'),
+                        (2, 'b', 'bb'),
+                        (3, 'c', 'cc'),
+                        (4, 'd', 'dd')
+                ) AS t (id, name, value)
+                """;
+        assertUpdate(format(createTableSql, target), 4);
```
Comment (djsstarburst) [inline testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java:5183]: Nit: Now that we have Java 17, we prefer `createTableSql.formatted(target)` to `String.format(createTableSql, targate)`.  Please use in all the new formatted output.

**Review by djsstarburst — APPROVED** [review]
LGTM.  I suggested replacing `String.format(format, ...)` calls with `format.formatted(...)`

File: testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java (hunk 1 above)
Comment (chenjian2664) [reply testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java:5183]: Got it, thanks

**Comment by chenjian2664** [comment]
@djsstarburst Could you help to merge this change?

**Comment by djsstarburst** [comment]
Hi @chenjian2664.  Unfortunately I don't have the privileges to merge in the trino repository.

@findepi or @losipiuk do have such privileges.

**Review by losipiuk — APPROVED** [review]

**Pull request merged**